### PR TITLE
chore: add rust guiding principles

### PR DESCRIPTION
**Pull request by FlakM**
Hi! I'd like to add this entry. Hope it's a correct section :+1:

### CODE REVIEW

File: draft/2024-11-27-this-week-in-rust.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ and just ask the editors to select the category.
 
 ### Observations/Thoughts
 
+* [Aligning with rust's Guiding Principle: my path to finding my 'Why' ](http://localhost:1313/posts/rust_guiding_principle/)
+
```
Comment (ekanna) [inline draft/2024-11-27-this-week-in-rust.md:43]: Url contains localhost!
localhost:1313

Comment (FlakM) [reply draft/2024-11-27-this-week-in-rust.md:43]: Oh that's embarrassing, sorry for that... That's what i get for using `conceallevel=2` :dagger:

File: draft/2024-11-27-this-week-in-rust.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ and just ask the editors to select the category.
 
 ### Observations/Thoughts
 
+* [Aligning with rust's Guiding Principle: my path to finding my 'Why' ](https://flakm.com/posts/rust_guiding_principle/)
```
Comment (mariannegoldin) [inline draft/2024-11-27-this-week-in-rust.md:42]:
```suggestion
* [Aligning with rust's Guiding Principle: my path to finding my 'Why'](https://flakm.com/posts/rust_guiding_principle/)
```

**Review by mariannegoldin — APPROVED** [review]
Thanks for the well-considered post!

**Pull request merged**